src/appkit.cs

```diff
@@ -23671,4 +23676,10 @@ interface NSOpenGLLayer
 		void Draw (NSOpenGLContext context, NSOpenGLPixelFormat pixelFormat, double t, ref CVTimeStamp ts);
 	}
 
+	[Protocol (IsInformal=true)]
```

Comment:
Ah, in that case I prefer `[Protocol (IsInformal=true)]`, because as you say `[Category]` is used to generate API that's called, not overridden.